heroku/cli
# refactor: migrate access:add to oclif/core

[Work item](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001lKBdpYAG/view)

Note: this PR also removes the orgs-v5 access:index file. I forgot to remove it in an earlier PR.

## Testing
- Check out this branch and run `yarn` and `yarn build`
- Add access to a team app
    - Choose a team app (I've been using k80-test on hk-front-end-online-test-team)
    - Run `./bin/run access:add EMAIL -a APP-NAME
- Add access to a personal app
    - Choose a personal (non-team) app
    - Choose a permissions level (deploy, manage, or operate)
    - run `./bin/run access:add EMAIL -a APP-NAME -p PERMISSIONS`
